packages/hive/configuration-schema.json

```diff
@@ -54,7 +54,7 @@
   "additionalProperties": true,
   "required": [
     "host",
-    "port",
+    "database",
```

Comment:
This looks out of scope - is it intended?

Reply:
It's kind related, There is an issue on docs for displaying only required properties in sample configuration. And hive was missing `database` in required properties 

[See the issue here](https://github.com/OpenFn/docs/issues/471)

Reply:
That's fine - so long as it's deliberate and expected. It should really be flagged in the main PR description, but you've clarified it now :)